client/webpack.storybook.config.js

```diff
@@ -37,6 +37,14 @@ config.module.loaders.push(
   }
 );
 
+config.externals = {
+  jsdom: 'window',
+  cheerio: 'window',
+  'react/lib/ExecutionEnvironment': true,
+  'react/lib/ReactContext': 'window',
```

Comment:
The way I understand this is that ReactContext can be required from window. The examples I found, treated this as pure external request ``` 'react/lib/ReactContext': true, ```. So, it probably works that way too.
https://webpack.github.io/docs/configuration.html#externals